src/extensions/correlograms/CrossCorrelogramsView/CrossCorrelogramsWidget.tsx

```diff
@@ -54,24 +50,22 @@ const CrossCorrelograms: React.FunctionComponent<SortingViewProps & SizeMeProps>
 
     const renderRow = ( pairs: {xkey: number, ykey: number}[], plotWidth: number ) => {
         return (
-            <Grid key={'range-'+pairs[0].ykey+'-to-'+pairs[pairs.length -1].ykey}>
+            <Grid key={'range-'+pairs[0].ykey+'-to-'+pairs[pairs.length -1].ykey} spacing={0}>
                 {
                     pairs.map((pair) => (
-                        <Grid key={pair.xkey + '-' + pair.ykey + '-' + plotWidth} item
-                                style={{ 'paddingBottom': '25px',
-                                    'marginBottom': '50px'}}>
+                        <Grid key={pair.xkey + '-' + pair.ykey + '-' + plotWidth} item>
                             <div
                             >
                                 <ClientSidePlot
-                                    dataFunctionName='fetch_correlogram_plot_data'
+                                    dataFunctionName='createjob_fetch_correlogram_plot_data'
                                     dataFunctionArgs={{
                                         sorting_object: sorting.sortingObject,
                                         unit_x: pair.xkey,
                                         unit_y: pair.ykey
                                     }}
                                     boxSize={{
                                         width: plotWidth,
-                                        height: plotWidth
+                                        height: plotWidth - 35
```

Comment:
We'll need to figure out how to do the plot box sizing in a better way. Here the "-35" is referring to vertical space taken up by labels. Will need to improve.